### PR TITLE
Issue #ED-3626 Angular migration from 14 to 15

### PR DESCRIPTION
**Pull request by mithun30052001**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,27 +11,27 @@
...
-    "@angular/cdk": "14.2.7",
-    "@angular/common": "^14.3.0",
-    "@angular/compiler": "^14.3.0",
-    "@angular/core": "^14.3.0",
-    "@angular/forms": "^14.3.0",
-    "@angular/material": "14.2.7",
-    "@angular/platform-browser": "^14.3.0",
-    "@angular/platform-browser-dynamic": "^14.3.0",
-    "@angular/router": "^14.3.0",
+    "@angular/animations": "^15.2.10",
```
Comment (princegupta1131) [inline package.json]: @mithun30052001 remove the ^,~ from package

**Review by princegupta1131 — COMMENTED** [review]
@mithun30052001 
update the version in library package.json 
update the readme file with latest version 
Raise a PR in this branch  https://github.com/Sunbird-Ed/sb-notification/tree/8.0.0_v15

File: package.json (hunk 1 above)
Comment (mithun30052001) [reply package.json]: @princegupta1131 The compatiblity symbols are removed

Comment (princegupta1131) [reply package.json]: @mithun30052001 
update the version in library package.json
update the readme file with the latest version
specified engines in the sb-notification package json Reference:  https://github.com/Sunbird-Ed/SunbirdEd-portal/blob/master/src/app/package.json#L30

Comment (mithun30052001) [reply package.json]: @princegupta1131  i have updated the package.json and readme and thereis no engines to update in sb-notification package.json

Comment (princegupta1131) [reply package.json]: @mithun30052001  please add  specified engines in the sb-notification package json Reference: https://github.com/Sunbird-Ed/SunbirdEd-portal/blob/master/src/app/package.json#L30

Comment (mithun30052001) [reply package.json]: Hi @princegupta1131 the engines are updated with node and npm versions

**Review by princegupta1131 — APPROVED** [review]

**Review by swayangjit — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -46,6 +46,10 @@
     "protractor": "~7.0.0",
     "ts-node": "~7.0.0",
     "tslint": "~6.1.0",
-    "typescript": "~4.6.4"
+    "typescript": "4.8.2"
+  },
+  "engines": {
```
Comment (princegupta1131) [inline package.json:51]: @mithun30052001    "engines": {
    "node": ">=16.13.0",
    "npm": ">=8.19.3"
  },

Comment (mithun30052001) [reply package.json:51]: @princegupta1131  changes updated

**Review by rajeevsathish — APPROVED** [review]

**Pull request merged**